### DISCUSSION TITLE
Hydreon RG15 malformed JSON string

### DISCUSSION THREAD

**Discussion by canis-ferrin** [arendst/Tasmota#21508] [Support request] [answered]
I flashed a Wemos D1 R2 with Tasmota 14.0.0.1, attached a RG15, seems to be working...
Web Interface: 
RG-15 Active	0.01 mm
RG-15 Event	0.08 mm
RG-15 Total	2.73 mm
RG-15 FlowRate	1.32 mm/h
Console:
12:07:38.293 MQT: Garten/RG15/tele/SENSOR = {"Time":"2024-05-27T12:07:38","RG-15":{"Active":, "Event":, "Total":, "FlowRate":}}
12:07:51.268 MQT: Garten/RG15/stat/RESULT = {"Sensor90":"R"}
12:08:02.296 MQT: Garten/RG15/tele/SENSOR = {"Time":"2024-05-27T12:08:02","RG-15":{"Active":, "Event":, "Total":, "FlowRate":}}
FHEM:
12:07:38.293 MQT: Garten/RG15/tele/SENSOR = {"Time":"2024-05-27T12:07:38","RG-15":{"Active":, "Event":, "Total":, "FlowRate":}}
12:07:51.268 MQT: Garten/RG15/stat/RESULT = {"Sensor90":"R"}
12:08:02.296 MQT: Garten/RG15/tele/SENSOR = {"Time":"2024-05-27T12:08:02","RG-15":{"Active":, "Event":, "Total":, "FlowRate":}}
So I can't get the data out.
Is this an issue in the Firmware or is something wrong?

**Comment by arendst** [comment]
Let me chk....

**Comment by arendst** [comment]
Verified!!  Nice find. Will fix now.

**Comment by arendst** [comment, marked as answer]
fixed in latest v14.0.0.2

**Comment by canis-ferrin** [comment]
Thanks for fast response, it's working now.
But: If I send a "SENSOR90 O" to the RG15 or type it in trhe console, the accumulation counter is not reset.
It looks like commands are not send to the sensor, but that's no problem.
